<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceLampHole</class>
 <widget class="QWidget" name="InterfaceLampHole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background:black;
}

QWidget#widget{
	border:1px solid #fefe00;
	border-radius:6px;	
}
QLabel#label_InterfaceLampHole{
	font:75 16px;
	color:#fefe00;
}

/*******页码******/
QLabel#label
{
	color:white;
	font :14px;
	border :1px solid white;
	border-radius:4px;
}
QLabel#label_pageNum{
	color:white;
	font:16px;
	border:1px solid white;
	border-top-left-radius:4px;
	border-bottom-left-radius:4px;
	border-right:none;
}
/*****总页数*****/
QLabel#label_totalPage{
	color:white;
	font:16px;
	border:1px solid white;
	border-bottom-right-radius:4px;
	border-top-right-radius:4px;
	border-left:none;
}

QPushButton{
	color:#f9f900;
	font:75 16px;
	border:1px solid  #fafa00;
	border-radius:6px;
}

QPushButton:pressed,QPushButton:focus{
	background:#0000fe;
}

QLineEdit#lineEdit_search{
	color: #fefe00;
}

QLabel#label_search{
	color: #fefe00;
}

QComboBox#cbx_enumSelection{
	font:75 14px;
	background:transparent;
	color:white;
	border:1px solid #fafa00;
	border-radius:4px;
}

QComboBox#cbx_enumSelection::down-arrow{
	image: url(:/ComboBox/Image/ComboBox/down_arrow.png);
}

QComboBox#cbx_enumSelection::drop-down{
	border:none;
	
}

QComboBox QAbstractItemView {
	font: 75 14px;
	background-color: #28231f;
	color: white;
	border:none;
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_InterfaceLampHole">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>LCU接口灯孔对应表</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>线号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>接口灯孔</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QComboBox" name="cbx_enumSelection">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>27</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>S</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>T</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Y</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbt_search">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbt_clear">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_pageNum">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_totalPage">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QPushButton" name="pbt_homePage">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>首页</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbt_endPage">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>尾页</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QPushButton" name="pbt_pageUp">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbt_pageDown">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
